src/style.css

```diff
@@ -0,0 +1,28 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
```

Comment:
```suggestion
* {
  box-sizing: border-box;
}
```
don't use anything except box-sizing inside `*` selector

Reply:
Why I cannot used it? it's not allowed under the terms.

Reply:
It's bad for performance, fix it